react/src/Toast/useToast.ts

```diff
@@ -36,7 +36,7 @@ export const useToast = ({
 
   const customToastImpl = useMemo(() => {
     const impl = ({
-      duration = 6000,
+      duration = initialProps.duration,
```

Comment:
Should we use initial duration props too for consistency? 

Should we also add back the default value to prevent duration changes due to this new prop assignment?

Reply:
Set the default value via param defaults. Had to rename it to `initialDuration`, similar to `initialStatus`
```
duration: initialDuration = 6000
```